websocket/src/websocket.cpp

```diff
@@ -67,6 +67,19 @@ const char* StateToString(State err)
 
 #undef STRING_CASE
 
+int dmStriCmp(const char* s1, const char* s2)
```

Comment:
We do have dmStrCaseCmp: https://defold.com/ref/stable/dmStringFunc/#dmStrCaseCmp:s1-s2

Could that be used instead?

Reply:
It could. We'll make a mental note of it and do that in another PR.